content/internal/procedure-version-control.md

```diff
@@ -205,12 +205,39 @@ Code should be thoroughly tested before a request is initiated, with tests condu
     + Restart your R session with `Ctrl`+`Shift`+`F10`. 
     
     
-* In Python when using the VSCode IDE:
+* In Python, when using the VSCode IDE:
```

Comment:
I'd prefer to keep this platform agnostic because there's no saying that a year down the line we're using another tool for writing Python. Hell you can use RStudio already.

Reply:
True true, I changed it to be platform agnostic but specific for notebooks.